.github/workflows/upload-tutorials-stats.yml

```
@@ -0,0 +1,40 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Upload tutorials stat
+
+# Controls when the workflow will run
+on:
+  schedule:
+  # Run this once per day.
+    - cron: "0 0 * * *"
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  get-tutorials-stats:
+    runs-on: ubuntu-latest
```

Let's make this job conditional on only it only running from the `pytorch/pytorch` repo, so that forked repos don't trigger it:

```suggestion
    runs-on: ubuntu-latest
    if: ${{ github.repository == 'pytorch/pytorch' }}
```

Example: https://github.com/pytorch/pytorch/blob/master/.github/workflows/docker-release.yml#LL32C9-L33C54